docs/unified/novacustom/recovery.md

```diff
@@ -47,6 +47,8 @@ The procedure will be slightly different depending on the model of your laptop.
       pitch pin header
     - USB-A to USB-B cable to connect the Arduino to the host
     - USB-C cable for grounding
+    
```

Comment:
@Stanislaw-bnk take a look at the `pre-commit` status. If failure from it is fixed, everything will be OK.